Samples/WindowsCopilotRuntime/cpp-console-unpackaged/CMakeLists.txt

```diff
@@ -0,0 +1,56 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+cmake_minimum_required(VERSION 3.10)
```

Comment:
I would generally recommend using 3.20 as your minimum IF you want to support customers with VS 2019 (16.11 was the last supported release).

Otherwise, 3.21 brings a few useful improvements and is the oldest version that shipped with VS 2022.